ansible/files/telemetry.sh

```diff
@@ -85,6 +85,9 @@ echo "export PYSPARK_SUBMIT_ARGS=\"--packages com.databricks:spark-csv_2.10:1.2.
 
 source $HOME/.bashrc
 
+# Setup Jupyter notebook
+aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/jupyter_notebook_config.py ~/.jupyter/jupyter_notebook_config.py
```

Comment:
Since it basically just contains the credentials already, we could also simply store `jupyter_notebook_config.py` in telemetry-spark-emr-2. Would that be a better option?